web/web/static/js/network_client.js

```diff
@@ -46,9 +57,9 @@ export const auth = {
     network: ky,    
     url: '/auth/',
 
-    delete: async (resource, data, successHandler, errorHandler) => {
+    post: async (resource, data, successHandler, errorHandler) => {
```

Comment:
The backend for this endpoint should be in the api layer as well. We're basically duplicating all this ky code here unnecessarily. If the backend code can't live under the api folder for some reason, then we should tweak this network client to accept a custom URL from the caller. But let's keep this DRY.

Reply:
I opted for moving the refresh token endpoint to live under the login api instead of removing the api/v1/ from the url in the ky code.